packages/core/__tests__/dlc/finance/ShortPut.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { HyperbolaPayoutCurvePiece } from '@node-dlc/messaging';
+import { expect } from 'chai';
+
+import { ShortPut } from '../../../lib/dlc/finance/ShortPut';
+import { HyperbolaPayoutCurve } from '../../../lib/dlc/HyperbolaPayoutCurve';
+
+describe('ShortPut', () => {
+  describe('1BTC-50k-base2-20digit curve', () => {
+    const strikePrice = BigInt(50000);
+    const contractSize = BigInt(1) ** BigInt(8);
```

Comment:
I think this is meant to be `const contractSize = BigInt(10) ** BigInt(8);`

Reply:
Good catch